parm/ccpp_v16beta_c96.nml.IN

```diff
@@ -48,8 +48,8 @@ deflate_level=1
   nudge_dz = .false.
   tau = 10.
   rf_cutoff = 7.5e2
-  d2_bg_k1 = 0.20
-  d2_bg_k2 = 0.0
+  d2_bg_k1 = 0.15
+  d2_bg_k2 = 0.02
```

Comment:
Do we need to update v16beta namelist? I thought it was fixed at Oct, 2019.

Reply:
Please see Ligia's comment below (https://github.com/ufs-community/ufs-weather-model/pull/149#pullrequestreview-433547405) and the discussion on the forum: https://forums.ufscommunity.org/threads/strength-second-order-damping-top-layers

Reply:
Just let you know that current GFSv16 forecast is using the updated coefficients for L127 levels, but GDAS forecast is using previous version.